web/src/components/LoginEmailCheck.tsx

```diff
@@ -0,0 +1,102 @@
+import { GenericTextField } from "@/components/GenericTextField";
+import { WithErrorBar } from "@/components/WithErrorBar";
+import { Heading } from "@/components/njwds-extended/Heading";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
+import { postUserEmailCheck } from "@/lib/api-client/apiClient";
+import { triggerSignIn } from "@/lib/auth/sessionHelper";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import analytics from "@/lib/utils/analytics";
+import { validateEmail } from "@/lib/utils/helpers";
+import { InputLabel } from "@mui/material";
+import Link from "next/link";
+
+import { useState, type ReactElement } from "react";
+
+export const LoginEmailCheck = (): ReactElement => {
+  const [email, setEmail] = useState<string>("");
+  const [emailError, setEmailError] = useState<string>("");
+
+  const handleSubmit = (email: string): void => {
+    const isValidEmail = validateEmail(email);
+
+    if (isValidEmail) {
+      checkEmailExists(email);
+    } else {
+      setEmailError(Config.checkAccountEmailPage.invalidEmailError);
+    }
+  };
+
+  const checkEmailExists = async (email: string): Promise<void> => {
+    try {
+      const response = await postUserEmailCheck(email);
+      if (response.found) {
+        setEmailError("");
+        triggerSignIn();
+        analytics.event.check_account_next_button.submit.go_to_myNJ_login();
+      }
+    } catch (error) {
+      if (error === 404) {
+        setEmailError(Config.checkAccountEmailPage.emailNotFoundError);
+      } else if (error === 500) {
+        setEmailError(Config.checkAccountEmailPage.serviceNotAvailableError);
+      } else {
+        setEmailError(Config.checkAccountEmailPage.defaultErrorMessage);
+      }
+    }
+  };
+
+  const { Config } = useConfig();
+
+  const handleTextInputChange = (value: string): void => {
+    setEmail(value);
+  };
+
+  return (
+    <div className="email-check-card padding-5 desktop:margin-x-2 radius-md">
+      <Heading level={1} styleVariant="h2">
+        {Config.checkAccountEmailPage.header}
+      </Heading>
+
+      <WithErrorBar hasError={Boolean(emailError)} type="ALWAYS">
+        <InputLabel htmlFor="email">Email</InputLabel>
+        <GenericTextField
+          inputWidth="default"
+          fieldName="email"
+          value={email}
+          handleChange={handleTextInputChange}
+          error={Boolean(emailError)}
+          validationText={emailError}
+        />
```

Comment:
[sand] In my testing, I found this field wouldn't submit if I hit the return key on my keyboard, only if I press the button. Can you confirm?

Reply:
Good catch. Added that [here](https://github.com/newjersey/navigator.business.nj.gov/pull/9276/files#diff-986f80750ed5eb242f707134a2ee52826c046cc700aede9eb614f795e5d83e85R56-R61) (plus a [test](https://github.com/newjersey/navigator.business.nj.gov/pull/9276/files#diff-8825d4c81eee8d76e01b6d3191340df0aa05b193e15a9cbc46af93a97d050a8fR85-R98)).